cli/src/pcluster/api/pcluster_api.py

```diff
@@ -121,6 +121,7 @@ def __init__(self, imagebuilder: ImageBuilder, image: ImageInfo):
         super().__init__(imagebuilder=imagebuilder)
         self.image_exist = True
         self.image_name = image.name
+        self.pcluster_image_id = image.pcluster_image_id
```

Comment:
if this is a class representing the PCluster Image, then the `self.image_id` must be the PCluster Image ID, while the EC2 AMI information needs to be moved under a subproperty